src/commands/rateSingleGame.ts

```diff
@@ -0,0 +1,166 @@
+import {
+  APIEmbed,
+  ButtonBuilder,
+  ButtonStyle,
+  ComponentType,
+  SlashCommandBuilder,
+} from "discord.js";
+
+import { Game } from "@prisma/client";
+
+import { Command, prisma } from "..";
+
+import { registerUser } from "../util/registerUser";
+
+const gameEmbedBuilder = (game: Game): APIEmbed => {
+  return {
+    description: game.description ?? undefined,
+    fields: [
+      {
+        inline: true,
+        name: "Number of players",
+        value: game.numPlayers.toString(),
+      },
+      { inline: true, name: "Is free?", value: game.isFree ? "Yes" : "No" },
+    ],
+    footer: { text: "Rate the game from 1 to 5." },
+    image: game.bannerImageURL ? { url: game.bannerImageURL } : undefined,
+    title: game.name,
+    thumbnail: game.thumbnailImageURL
+      ? { url: game.thumbnailImageURL }
+      : undefined,
+    url: game.gameURL ?? undefined,
+  };
+};
+
+const builder = new SlashCommandBuilder()
+  .setName("ratesinglegame")
+  .setDescription("Rate a single game for your current user.");
+
+builder.addStringOption((option) =>
+  option
+    .setName("name")
+    .setDescription("The name of the game to re-rate.")
+    .setRequired(true)
+    .setAutocomplete(true)
+);
+
+export const rateSingleGame: Command = {
+  builder,
+  execute: async (interaction) => {
+    const guildId = interaction.guildId;
+    if (!guildId) {
+      await interaction.reply("This command can only be used in a server.");
+      return;
+    }
+
+    const user = await registerUser(interaction.user);
+
+    const name = await interaction.options.getString("name");
+    if (name == null) {
+      await interaction.reply("Game not found.");
+      return;
+    }
```

Comment:
Typescript has implicit "falsiness" to it's values, which can be useful to use here so we can check all cases like `undefined`, `null`, or otherwise "false":
```suggestion
    if (!name) return;
```
It's also reasonable I think to ignore replying to the user in this case since they can't actually type this command without providing a name. If this ever happens, idk how the user even got here.